### PR TITLE
fix failing Composer test

### LINKED ISSUE #9114: composer.2022_airflow_summit.data_analytics_process_test: test_process failed
Note: #8633 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: 234c2cdc66d552f7c40d70ee3db2c21ffec50825
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/ceb4c78d-5cf4-4688-8812-9a49b4a8521f), [Sponge](http://sponge2/ceb4c78d-5cf4-4688-8812-9a49b4a8521f)
status: failed
<details><summary>Test output</summary><br><pre>Traceback (most recent call last):
  File "/workspace/composer/2022_airflow_summit/data_analytics_process_test.py", line 109, in test_dataproc_batch
    raise (e)
  File "/workspace/composer/2022_airflow_summit/data_analytics_process_test.py", line 88, in test_dataproc_batch
    response = operation.result()
  File "/workspace/composer/2022_airflow_summit/.nox/py-3-8/lib/python3.8/site-packages/google/api_core/future/polling.py", line 137, in result
    raise self._exception
google.api_core.exceptions.Aborted: 409 Job failed with message [Exception: Error reading data_analytics_process_test_cf0c0148_3907_480b_b539_27020d4cdd73.data_analytics_process_test_joined_cf0c0148_3907_480b_b539_27020d4cdd73]. Additional details can be found at:
https://console.cloud.google.com/dataproc/batches/northamerica-northeast1/summit-dag-test-cf0c0148-3907-480b-b539-27020d4cdd73?project=python-docs-samples-tests
gcloud dataproc batches wait 'summit-dag-test-cf0c0148-3907-480b-b539-27020d4cdd73' --region 'northamerica-northeast1' --project 'python-docs-samples-tests'
https://console.cloud.google.com/storage/browser/dataproc-staging-na-northeast1-1012616486416-nj3uvik7/google-cloud-dataproc-metainfo/6a662b7b-f350-431f-940f-7b86713b3e1f/jobs/srvls-batch-42ee8c3e-237b-446c-a1b2-84441a0bdde6/
gs://dataproc-staging-na-northeast1-1012616486416-nj3uvik7/google-cloud-dataproc-metainfo/6a662b7b-f350-431f-940f-7b86713b3e1f/jobs/srvls-batch-42ee8c3e-237b-446c-a1b2-84441a0bdde6/driveroutput</pre></details>

### PR DESCRIPTION
**Pull request by leahecole**
## Description

@bradmiro helped me fix the test failures here by removing a deprecated parameter with a more reliable one. I also fixed a mistake in a BQ query and changed the DAG name in the expansion file so it didn't have the same DAG name as the other one

Fixes #9114

DAG run for data_analytics_dag - 
image

data_analytics_dag_expansion run
image

### CODE REVIEW

**Comment by leahecole** [comment]
Python 3.11 failures are currently expected

File: composer/2022_airflow_summit/data_analytics_dag.py [hunk 1]
```diff
@@ -33,14 +33,14 @@
...
 # Dataproc configs
 BUCKET_NAME = "{{var.value.gcs_bucket}}"
-PYSPARK_JAR = "gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
 PROCESSING_PYTHON_FILE = f"gs://{BUCKET_NAME}/data_analytics_process.py"
 
-
 BATCH_ID = "data-processing-{{ ts_nodash | lower}}"  # Dataproc serverless only allows lowercase characters
 BATCH_CONFIG = {
+    "runtime_config": {
+        "version": "1.1"  # Will need to be changed to 2.0+ when that's GA
```
Comment (bradmiro) [inline composer/2022_airflow_summit/data_analytics_dag.py:41]: Technically 2.0 is already GA but doesn't yet support the Spark-BigQuery connector. 1.1 is LTS until August 2025 so I think you can safely remove this comment or mention it should be revisited then.

File: composer/2022_airflow_summit/data_analytics_dag_expansion.py [hunk 2]
```diff
@@ -37,13 +37,14 @@
 
 # Dataproc configs
 BUCKET_NAME = "{{var.value.gcs_bucket}}"
-PYSPARK_JAR = "gs://spark-lib/bigquery/spark-bigquery-with-dependencies_2.12-0.26.0.jar"
 PROCESSING_PYTHON_FILE = f"gs://{BUCKET_NAME}/data_analytics_process_expansion.py"
 
 BATCH_ID = "data-processing-{{ ts_nodash | lower}}"  # Dataproc serverless only allows lowercase characters
 BATCH_CONFIG = {
+    "runtime_config": {
+        "version": "1.1"  # Will need to be changed to 2.0+ when that's GA
```
Comment (bradmiro) [inline composer/2022_airflow_summit/data_analytics_dag_expansion.py:45]: See prior comment.

File: composer/2022_airflow_summit/data_analytics_process_expansion_test.py [hunk 3]
```diff
@@ -75,8 +74,10 @@ def test_dataproc_batch(test_bucket, bq_dataset):
 
     BATCH_ID = f"summit-dag-expansion-test-{TEST_ID}"  # Dataproc serverless only allows lowercase characters
     BATCH_CONFIG = {
+        "runtime_config": {
+            "version": "1.1"  # Will need to be upgraded to 2.0+ when that is GA
```
Comment (bradmiro) [inline composer/2022_airflow_summit/data_analytics_process_expansion_test.py:78]: See prior comment.

File: composer/2022_airflow_summit/data_analytics_process_test.py [hunk 4]
```diff
@@ -57,8 +56,10 @@ def test_dataproc_batch(test_bucket, bq_dataset):
 
     BATCH_ID = f"summit-dag-test-{TEST_ID}"  # Dataproc serverless only allows lowercase characters
     BATCH_CONFIG = {
+        "runtime_config": {
+            "version": "1.1"  # will need to be updated to 2.0+ after its release
```
Comment (bradmiro) [inline composer/2022_airflow_summit/data_analytics_process_test.py:60]: See prior comment.

File: composer/2022_airflow_summit/data_analytics_dag.py (hunk 1 above)
Comment (leahecole) [inline composer/2022_airflow_summit/data_analytics_dag.py:41]:
```suggestion
        "version": "1.1"
```

File: composer/2022_airflow_summit/data_analytics_dag_expansion.py (hunk 2 above)
Comment (leahecole) [inline composer/2022_airflow_summit/data_analytics_dag_expansion.py:45]:
```suggestion
        "version": "1.1"
```

File: composer/2022_airflow_summit/data_analytics_process_expansion_test.py (hunk 3 above)
Comment (leahecole) [inline composer/2022_airflow_summit/data_analytics_process_expansion_test.py:78]:
```suggestion
            "version": "1.1"
```

File: composer/2022_airflow_summit/data_analytics_process_test.py (hunk 4 above)
Comment (leahecole) [inline composer/2022_airflow_summit/data_analytics_process_test.py:60]:
```suggestion
            "version": "1.1"
```

**Review by nicain — APPROVED** [review]

**Pull request merged**